readme.md

```diff
@@ -574,6 +574,7 @@ Non-synced local preferences are stored within `Alfred.alfredpreferences` under
 - [alfred-npm-versions](https://github.com/mrmartineau/alfred-npm-versions) - Lookup the latest 15 versions for an npm package
 - [alfred-travis-ci](https://github.com/adriantombu/alfred-travis-ci) - Check the status of your Travis CI builds
 - [alfred-github-trending](https://github.com/mikqi/alfred-github-trending) - Search trending repositories on GitHub
+- [alfred-elm](https://github.com/nicklayb/alfred-elm) - Browser Elm packages documentation
```

Comment:
I think this reads a bit better?

```suggestion
- [alfred-elm](https://github.com/nicklayb/alfred-elm) - Documentation for browser Elm packages
```

Reply:
In fact I made a typo 🤦🏼‍♂️ It « Browse », not « Browser », it makes more sense without mistake ahah, I’ll fix this today